soc: align CONFIG_SOC values to match soc.yml names

tejlmand opened the pull request:
This commit aligns CONFIG_SOC values for Nordic nRF SoCs to the names defined in soc.yml.

tejlmand committed:
soc: align CONFIG_SOC values to match soc.yml names

This commit aligns CONFIG_SOC values for Nordic nRF SoCs to the names
defined in soc.yml.

Signed-off-by: Torsten Rasmussen <Torsten.Rasmussen@nordicsemi.no>

nordicjm commented on soc/nordic_nrf/nrf51/Kconfig.soc:
```diff
@@ -21,6 +21,6 @@ config SOC_NRF51822_QFAC
-	default "nRF51822_QFAA" if SOC_NRF51822_QFAA
-	default "nRF51822_QFAB" if SOC_NRF51822_QFAB
-	default "nRF51822_QFAC" if SOC_NRF51822_QFAC
+	default "nrf51822" if SOC_NRF51822_QFAA
+	default "nrf51822" if SOC_NRF51822_QFAB
+	default "nrf51822" if SOC_NRF51822_QFAC
```
```suggestion
	default "nrf51822" if SOC_NRF51822_QFAA || SOC_NRF51822_QFAB || SOC_NRF51822_QFAC
```

tejlmand replied:
Done

nordicjm commented on soc/nordic_nrf/nrf52/Kconfig.soc:
```diff
@@ -81,14 +81,14 @@ config SOC_NRF52840_QIAA
+	default "nrf52810" if SOC_NRF52810_QFAA
+	default "nrf52811" if SOC_NRF52811_QFAA
+	default "nrf52820" if SOC_NRF52820_QDAA
+	default "nrf52832" if SOC_NRF52832_CIAA
+	default "nrf52832" if SOC_NRF52832_QFAA
+	default "nrf52832" if SOC_NRF52832_QFAB
```
```suggestion
	default "nrf52832" if SOC_NRF52832_CIAA || SOC_NRF52832_QFAA || SOC_NRF52832_QFAB
```

tejlmand replied:
Done

nordicjm commented on soc/nordic_nrf/nrf52/Kconfig.soc:
```diff
@@ -81,14 +81,14 @@ config SOC_NRF52840_QIAA
+	default "nrf52832" if SOC_NRF52832_CIAA
+	default "nrf52832" if SOC_NRF52832_QFAA
+	default "nrf52832" if SOC_NRF52832_QFAB
+	default "nrf52833" if SOC_NRF52833_QDAA
+	default "nrf52833" if SOC_NRF52833_QIAA
```
```suggestion
	default "nrf52833" if SOC_NRF52833_QDAA || SOC_NRF52833_QIAA
```

tejlmand replied:
Done

nordicjm commented on soc/nordic_nrf/nrf52/Kconfig.soc:
```diff
@@ -81,14 +81,14 @@ config SOC_NRF52840_QIAA
+	default "nrf52832" if SOC_NRF52832_QFAB
+	default "nrf52833" if SOC_NRF52833_QDAA
+	default "nrf52833" if SOC_NRF52833_QIAA
+	default "nrf52840" if SOC_NRF52840_QFAA
+	default "nrf52840" if SOC_NRF52840_QIAA
```
```suggestion
	default "nrf52840" if SOC_NRF52840_QFAA || SOC_NRF52840_QIAA
```

tejlmand replied:
Done

nordicjm commented on soc/nordic_nrf/nrf53/Kconfig.soc:
```diff
@@ -26,5 +26,5 @@ config SOC_NRF5340_CPUNET_QKAA
 config SOC
-	default "nRF5340_CPUAPP_QKAA" if SOC_NRF5340_CPUAPP_QKAA
-	default "nRF5340_CPUNET_QKAA" if SOC_NRF5340_CPUNET_QKAA
+	default "nrf5340" if SOC_NRF5340_CPUAPP_QKAA
+	default "nrf5340" if SOC_NRF5340_CPUNET_QKAA
```
```suggestion
	default "nrf5340" if SOC_NRF5340_CPUAPP_QKAA || SOC_NRF5340_CPUNET_QKAA
```

tejlmand replied:
Done

tejlmand committed:
boards: nrf51dk/dongle: change SoC to nRF51822

The physical nRF51dk / nRF51dongle hardware contains a nRF51422 SoC.
In Zephyr, only the nRF51822 SoC, is implemented.

In Zephyr, the nRF51422 SoC is build as a nRF51822, therefore use the
nRF51822 SoC model for those kits instead on the non-existing nRF51422.

Signed-off-by: Torsten Rasmussen <Torsten.Rasmussen@nordicsemi.no>
